src/plan/tracing.rs

```diff
@@ -74,7 +74,7 @@ impl ObjectQueue for VectorQueue<ObjectReference> {
 /// A transitive closure visitor to collect all the edges of an object.
 pub struct ObjectsClosure<'a, E: ProcessEdgesWork> {
     buffer: VectorQueue<EdgeOf<E>>,
-    worker: &'a mut GCWorker<E::VM>,
+    pub(crate) worker: &'a mut GCWorker<E::VM>,
```

Comment:
This no longer needs to be `pub(crate)`.

Reply:
I think we still need this. When we do `worker.shared.increase_live_bytes()` in `ScanObjectsWork::do_work_common()`, we already created `ObjectsClosure` which takes `&mut GCWorker`. We can't use `worker` directly, and we have to use it through `ObjectsClosure`.

Reply:
Oh, sorry. I didn't notice that.